src/Fixer/ClassNotation/OrderedClassElementsFixer.php

```diff
@@ -345,8 +345,6 @@ private function getElements(Tokens $tokens, $startIndex)
             $elements[] = $element;
             $startIndex = $element['end'] + 1;
         }
-
-        return [];
```

Comment:
This is needed, for example, if the break on line 342 is executed.

Reply:
The `break` statement is for the `for` loop. The only way to exit the outer `while` is the `return` statement at line 307.

Reply:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/commit/93c04b47699de6cf64c0252994403fad14ef9c20

Reply:
// cc @SpacePossum